app/views/users/registrations/new.html.erb

```diff
@@ -20,11 +20,3 @@
     </div>
   </div>
 </div>
-
-<script>
```

Comment:
We need to keep this script, otherwise users will double click on the Signup button and see an error message saying an account already exists.